CHANGELOG.md

```diff
@@ -1,3 +1,62 @@
+# 0.3.0 (TBD)
```

Comment:
Technically this module doesn't have breaking changes but the underlying `node-mssql` does. It might be worth finally going to `1.0.0` to properly indicate the breaking change so major can be used to indicate breaking changes in the future. Thoughts @gshively11?

Reply:
for `0.x.x` versions, the minor is technically a "major" bump right?  e.g. people with `^0.2.0` won't automatically get `0.3.0`.  i'm not against going to `1.0.0`, but i'd rather do that when i've confirmed AGs work as intended

Reply:
Pre `1.0.0` you are safe from `^` bumping a major version in semver since `^ === ~` until you hit `1.0.0`. People tend to limit the time in pre `1.0.0` land due to the inconsistent semantics though. I think its fair to confirm AGs work as you expect and then pull the trigger. Just wanted to encourage getting to `1.0.0` since it is production ready software :).

Reply:
^ you make a good point, will bump for the next release